0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/File.scala

```diff
@@ -167,7 +167,14 @@ object File {
               val gapErrors = Common.checkSchemasConsistency(files.map(_.content.self)).leftMap(_.map(error => Error.ConsistencyError(error)))
               Ior.fromEither(gapErrors).putRight(files)
           }
-          checked.leftMap(_.concat(jsonSchemas.collect { case Left(e) => e }))
+
+          val errors = NonEmptyList.fromList(jsonSchemas.collect { case Left(e) => e })
+
+          (checked, errors) match {
+            case (Ior.Left(e), Some(e2)) => Ior.Left(e.concatNel(e2))
+            case (_, Some(e)) => Ior.Left(e)
```

Comment:
Shouldn't we check `Ior.Both(e, _), Some(e)` case specifically?

Reply:
Yes, we should check it too. I will try to combine them with one of the Cats operator.